### DISCUSSION TITLE
Ferramenta Gestão para Organizadores de Evento

### DISCUSSION THREAD

**Discussion by keppelen** [frontendbr/forum#650] [Discussão]
Quem organiza eventos na area de TI e quer participar na criação de um projeto para organizar as finanças do evento?

O Projeto já está em andamento, falta pouca coisa para que mais pessoas possam usar e dando feedback, o Front in BH já esta usando no dia a dia :)

**Comment by yanmagale** [comment]
Sensacional a ideia @keppelen , ter uma ferramenta para gerenciar os gastos é muito importante para a organização e realização do evento.

 Conte comigo para ajudar :smile:
